.env

```diff
@@ -1,2 +0,0 @@
-export DATALAD_REGISTRY_INSTANCE_PATH=$PWD/instance
-export DATALAD_REGISTRY_DB_INSTANCE_PATH=$PWD/instance/db
```

Comment:
could you please complement this removal with providing a file (or files) instead of this which would setup system in one of those configurations.

Reply:
`template.env.dev` and `template.env.testing` are now provided. They are to be used with `docker-compose.dev.yml` and `docker-compose.testing.yml` respectively.